app/client/graphQL/types.ts

```diff
@@ -0,0 +1,338 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {CustomStatusDuration} from '@constants';
```

Comment:
```suggestion

import {CustomStatusDuration} from '@constants';
```

I wonder if there is a way to auto generate this types.

Reply:
Hopefully there is, but we will address that while improving the developer experience.